data/gen.go

```diff
@@ -251,13 +310,19 @@ func main() {
 		height := parseInt(record[3])
 		weight := parseInt(record[4])
 		baseexp := parseInt(record[5])
-		pokemon = append(pokemon, data_pokemon{
+		dp := data_pokemon{
 			Identifier:     record[1],
 			SpeciesId:      species_id,
 			Height:         height,
 			Weight:         weight,
 			BaseExperience: baseexp,
-		})
+		}
+		if species_id == 94 { // gengar
+			// HACK: The data says that gengar has the ability levitate, which is not available in gen 4. There is also no other ability set for gengar.
```

Comment:
I mentioned in the GBM that Levitate was introduced in Gen III: https://bulbapedia.bulbagarden.net/wiki/Levitate_(Ability)